### PR TITLE
Feat/advanced doc

### PR DESCRIPTION
**Pull request by alexgeron-Luos**
⚠ PLEASE DO NOT DELETE THE TEXT BELOW ⚠

## Pull request's description

Add a new section in Luos documentation: Advanced documentation

## Checklist before merging (please do not edit)
You must review you work before to submit the review to others.

### Self-review of your content
Remember the content must be readable and understandable by someone else than yourself.
- [x] From a technical point of view.
- [x] From a grammatical point of view (spelling mistakes, typos, clarity, etc.), and following the guidelines at the bottom.

### External reviews of your content
- [x] You requested a technical review.
- [x] You requested a grammatical review.
- [ ] You validated with @K0rdan or @alexgeron-Luos that it is safe to merge.

### Some guidelines to keep in mind
- Colons (:), semi-colons (;), exclamation (!), and interrogation points (?) are not preceded by a space (like full stops and commas). E.g.: Colons!
- File names and/or address are put in *italic*. E.g.: The file _main.c_.
- Functions, variables, or more generally short codes are put between grave accents. E.g.: To obtain `code()`, type \`code()\`.
- Long codes are put into blocks of code with three grave accent on each side, and the language's name:<br />
\`\`\`c<br />
// Some C language <br />
\`\`\`<br />

- "Luos engine" has a upper case on the **L** for "Luos" and lower case on the **e** for "engine".
- We call it "Luos engine" as a proper name, and ***not*** "the Luos engine".
- Following that fashion, anything that's owned by "Luos engine" implies that we must use `'s` as the standard English rule to show the possessive case, e.g. "Luos engine's API".
- The names pipe, gate, inspector, sniffer, app or application, driver, etc. have a lower case and can be refereed with the determiner `the`. E.g.: The inspector.

### CODE REVIEW

**Comment by Simonbdy** [comment]
@alexgeron-Luos can you please provide a description in the part "Pull request's description"? Thanks

**Review by K0rdan — CHANGES_REQUESTED** [review]

File: docs/advanced-doc/index.mdx [hunk 1]
```diff
@@ -0,0 +1,15 @@
...
+custom_edit_url: null
+title: 'Luos Advanced documentation'
+---
+
+import ToolsCard from '@site/src/components/tools/ToolsCard.js';
+import dataJson from '@site/src/components/tools/data/advancedDoc/dataAdvancedDoc.json';
+
+# Luos Advanced documentation
+
+Change desc
```
Comment (K0rdan) [inline docs/advanced-doc/index.mdx:13]: To remove or update

**Review by Simonbdy — CHANGES_REQUESTED** [review]
@alexgeron-Luos can you please provide a description in the part "Pull request's description"? Thanks

**Review by K0rdan — APPROVED** [review]

**Review by Simonbdy — CHANGES_REQUESTED** [review]
I fixed a few typos.

There are some typos to fix on the images as well: 

static/assets/images/advanced-doc/robus/interface/robus-4-3-dark.svg
static/assets/images/advanced-doc/robus/interface/robus-4-3-light.svg
-> Change `receive pending` to `Reception pending`

static/assets/images/advanced-doc/robus/interface/robus-4-3-dark.svg
static/assets/images/advanced-doc/robus/protocol/robus-3-3-light.svg
-> Change `CRC calculate` to `CRC calculated` and `CRC receive` to `CRC received` 

static/assets/images/advanced-doc/robus/topology/robus-2-3-dark.svg
static/assets/images/advanced-doc/robus/topology/robus-2-3-light.svg
-> Change `anothe` to `another`

After that it will be good for me to merge

**Review by Simonbdy — APPROVED** [review]
Ok for me

**Pull request merged**